src/test/groovy/cz/actis/tutorial/testers/Tester5Spec.groovy

```diff
@@ -0,0 +1,18 @@
+package cz.actis.tutorial.testers
+
+import cz.actis.tutorial.testers.tester4.Point
+import cz.actis.tutorial.testers.tester5.Line2D
+import spock.lang.Specification
+
+class Tester5Spec extends Specification {
+    def "is perpendicular test"() {
```

Comment:
Stejnyum zpusobem, jako jsem psal `is perpendicular test` udelej test na velikost uhlu.
Pomuze ti, kdyz si ty body nakresis.